README.md

```diff
@@ -1222,10 +1220,79 @@ seq(ap(parx, parf)).value(console.log);
 //Or concurrent sequencing
 seq(sequence(Par, [parx, parf])).value(console.log);
 //> [x, f]
+```
+
+##### Par
+
+Converts a Future to a ConcurrentFuture
+
+<details><summary><code>Par :: Future a b -> ConcurrentFuture a b</code></summary>
+
+```hs
+Par :: Future a b -> ConcurrentFuture a b
+```
+
+</details>
+
+##### Par.of
+
+Construct a ConcurrentFuture with the given resolution value.
```

Comment:
You're using _Construct_ here but _Convert**s**_ in the section above. You should either add an _s_ here or drop the _s_ in the section above.